source/fundamentals/crud/read-operations.txt

```diff
@@ -2,7 +2,9 @@
 Read Operations
 ===============
 
-.. default-domain:: mongodb
+.. meta::
+   :description: Learn the concepts and commands for MongoDB read operations by using the {+driver-long+}
+related to retrieving data from MongoDB by using the {+driver-long+}.
```

Comment:
I: this seems like a run-on sentence. The driver is mentioned twice, which I think isn't necessary.